src/azure-cli/azure/cli/command_modules/appconfig/feature.py

```diff
@@ -1259,10 +1263,11 @@ def __list_all_keyvalues(azconfig_client,
     if unescaped_comma_regex.search(key_filter):
         raise CLIError("Comma separated feature names are not supported. Please provide escaped string if your feature name contains comma. \nSee \"az appconfig feature list -h\" for correct usage.")
 
-    label = prep_label_filter_for_url_encoding(label)
+    label = prep_filter_for_url_encoding(label)
+    prepped_tags = prep_filter_for_url_encoding(tags)
```

Comment:
Do we need to do this for each tag instead?

Reply:
`prepped_tags = [prep_filter_for_url_encoding(tag) for tag in tags] if tags else []`